db/migrate/20210615101632_add_dob_and_trn_to_users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddDobAndTrnToUsers < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:users, bulk: true) do |t|
+      t.date :date_of_birth, null: true
+      t.text :teacher_reference_number, null: true
+    end
+  end
```

Comment:
I don't think those should live on `users` table - we have admins, induction coordinators, lead providers there.

Should we make a new profile for this?

Reply:
ok, i'll remove all the extra stuff. the endpoint will be just to create a user. i'll do another PR to be able to create a npq profile with all the stuff i just removed (plus there will be other stuff not present here)